src/app.js

```diff
@@ -1,83 +1,145 @@
-const todoArr = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Complete To Do list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Manage all your lists in one place',
-    completed: false,
-    index: 2,
-  },
-];
+const todoArr = [];
+/* eslint-disable no-use-before-define */
+const add = () => {
+  const list = document.querySelector('.task-list');
+  const itemsLocal = JSON.parse(localStorage.getItem('itemsLocal'));
+  todoArr.splice(0, todoArr.length, ...itemsLocal);
```

Comment:
- [ ] When I start the app for the first time I see an error, this error is shown because `itemsLocal` does not exist yet, I will recommend you to have something like this:
```javascript
  const itemsLocal = JSON.parse(localStorage.getItem('itemsLocal')) || [];
```
- So, if `itemsLocal` does not exist it will be an empty array, this way you don't need `todoArr`, and you can work directly with `itemsLocal`
- Screenshot of the error:
    ![imagen](https://user-images.githubusercontent.com/60789317/145607267-b286215b-52ee-4967-8581-9f93b97daf26.png)